examples/backends/vllm/launch/xpu/disagg_multimodal_epd_xpu.sh

```
@@ -127,7 +127,7 @@ fi
 echo "Starting encode worker on GPU $DYN_ENCODE_WORKER_GPU (GPU mem: $DYN_ENCODE_GPU_MEM)..."
 VLLM_NIXL_SIDE_CHANNEL_PORT=20097 \
 env $DEVICE_AFFINITY_ENV=$DYN_ENCODE_WORKER_GPU \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)